src/ReleaseHistory.md

```diff
@@ -358,3 +358,4 @@
 * API BREAKING: `threadFlowLocation.executionOrder` can be 0 and defaults to -1, so minimum: -1, default: -1
 * API BREAKING: Rename artifact role `traceFile` to `tracedFile`.
 * API NON-BREAKING: Add artifact role `debugOutputFile`.
+* API BREAKING: `notification.physicalLocation`�of type `physicalLocation`�changed to `notification.locations`�of type `locations`.
```

Comment:
>� [](start = 47, length = 1)

What are these Unicode characters?

Reply:
:O

---
In reply to: [278289161](https://github.com/Microsoft/sarif-sdk/pull/1456#discussion_r278289161) [](ancestors = 278289161)